src/vivarium/framework/randomness.py

```diff
@@ -471,22 +471,6 @@ def get_draw(self, index: Index, additional_key: Any = None) -> pd.Series:
 
         return draw
 
-    def get_seed(self, additional_key: Any = None) -> int:
```

Comment:
Is it better to deprecate rather than remove this?

Reply:
In theory, yes.  We don't have a deprecation strategy (and is one of the reasons we're not at a v1 release).  So don't worry about it for now.

Reply:
https://jira.ihme.washington.edu/browse/MIC-1614